src/Navbar/NavbarSmall.js

```diff
@@ -0,0 +1,47 @@
+import { NavbarRespBig, NavbarRespSmall, NavbarToggler, DarkMode, NavbarHidden } from "./Navbar.style"
+import { Link } from "react-router-dom"
+import SearchBar from "./SearchBar"
+import CurrencyDropDown from "./CurrencyDropDown"
+import { useEffect, useState } from "react"
+import { useWindowSize } from "../helpers/useWindowSize"
+
+function NavbarSmall() {
+    // UseState Vars
+    const [hidden, setHidden] = useState(true)
+    // Getting the current window size with custom hook
+    const size = useWindowSize()
+    // Whenever window size changes, check and adjust for size
+    useEffect(() => {
+        if (size[0] > 1000 && hidden === false) {
+            setHidden(false)
+        }
+    }, [useWindowSize()])
```

Comment:
yoooo, this is not the right way to do this, I can;t believe react is not throwing an error


`useEffect(() => {
        if (size[0] > 1000 && hidden === false) {
            setHidden(false)
        }
    }, [size])`